token-lending/program/tests/set_lending_market_owner.rs

```diff
@@ -0,0 +1,139 @@
+mod helpers;
+
+use helpers::*;
+use solana_program::instruction::{AccountMeta, Instruction};
+use solana_program_test::*;
+use solana_sdk::{
+    instruction::InstructionError,
+    pubkey::Pubkey,
+    signature::{Keypair, Signer},
+    transaction::{Transaction, TransactionError},
+};
+use spl_token_lending::{
+    error::LendingError,
+    instruction::{set_lending_market_owner, LendingInstruction},
+    processor::process_instruction,
+};
+
+#[tokio::test]
+async fn test_success() {
+    let mut test = ProgramTest::new(
+        "spl_token_lending",
+        spl_token_lending::id(),
+        processor!(process_instruction),
+    );
+
+    // limit to track compute unit increase
+    test.set_bpf_compute_max_units(4_000);
+
+    let usdc_mint = add_usdc_mint(&mut test);
+    let lending_market = add_lending_market(&mut test, usdc_mint.pubkey);
+    let (mut banks_client, payer, recent_blockhash) = test.start().await;
+
+    let new_owner = Pubkey::new_unique();
+    let mut transaction = Transaction::new_with_payer(
+        &[set_lending_market_owner(
+            spl_token_lending::id(),
+            lending_market.pubkey,
+            lending_market.owner.pubkey(),
+            new_owner,
+        )],
+        Some(&payer.pubkey()),
+    );
+
+    transaction.sign(&[&payer, &lending_market.owner], recent_blockhash);
+
+    banks_client
+        .process_transaction(transaction)
+        .await
+        .map_err(|e| e.unwrap())
+        .unwrap();
```

Comment:
What does this do if no error is expected / what's the difference between this and

```rust
assert!(banks_client.process_transaction(transaction).await.is_ok());
```

Reply:
If it errors, it will panic due to the unwrap. An assert is better, thanks!

Reply:
Oops, my patch for the assert didn't get pushed before the auto-merge label took effect. I'll clean this up later, promise 😉